misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt

```diff
@@ -0,0 +1,60 @@
+package misk.redis
+
+import misk.time.FakeClock
+import java.time.Duration
+import java.time.Instant
+import java.util.concurrent.ConcurrentHashMap
+import javax.inject.Inject
+
+class FakeRedis : Redis {
+  @Inject lateinit var clock: FakeClock
+
+  // The value type stored in our key-value store
+  private data class Value (
+    val value: String,
+    val expiryInstant: Instant
+  )
+
+  // Acts as the Redis key-value store
+  private val keyValueStore = ConcurrentHashMap<String, Value>()
+
+  override fun del(key: String): Long {
+    if (!keyValueStore.containsKey(key)) return 0
+
+    keyValueStore.remove(key)
+    return 1
+  }
+
+  override fun del(vararg keys: String): Long {
```

Comment:
`Long` here is a bit weird - unlikely there will be greater than 2^32 keys, and `Int` doesn't require typecasting in most cases.

Reply:
Agreed, changed to `Int`